models/repo/archiver.go

```diff
@@ -65,6 +68,31 @@ func GetRepoArchiver(ctx context.Context, repoID int64, tp git.ArchiveType, comm
 	return nil, nil
 }
 
+// ExistsRepoArchiverWithStoragePath checks if there is a RepoArchiver for a given storage path
+func ExistsRepoArchiverWithStoragePath(ctx context.Context, pth string) (bool, error) {
```

Comment:
What happened to the `a` in `path`?

Reply:
`path` is a module name within the std go-library. Naming the variable `path` would shadow that module and would cause problems in future if that module needs to be imported.

Reply:
why not `storage_path`? I know its longer but it doesnt look so weird